docs/ref/models/database-functions.txt

```diff
@@ -428,6 +429,12 @@ Usage example::
 
     .. attribute:: lookup_name = 'week'
 
+.. class:: ExtractQuarter(expression, tzinfo=None, **extra)
```

Comment:
Add to the examples below? I guess we're also missing ExtractWeek in there. Could be added in a separate commit and backported.